Bigtable/src/DataClient.php

```diff
@@ -241,6 +241,7 @@ public function upsert(array $rows, array $options = [])
      *           associative array which may contain a start key
      *           (`startKeyClosed` or `startKeyOpen`) and/or an end key
      *           (`endKeyOpen` or `endKeyClosed`).
+     *     @type filter $filter A {@see Google\Cloud\Bigtable\Filter\FilterInterface} object to filter the rows.
```

Comment:
The type here can be `FilterInterface`, and the class will need to include the import :) (this will help the information display nicely on our documentation site). Also, what do you think of something like the following for the description?

```A filter used to take an input row and produce an alternate view of the row based on the specified rules. To learn more please see {@see Google\Cloud\Bigtable\Filter} which provides static factory methods for the various filter types.```

Reply:
will update it. Thanks.

Reply:
updated